src/app/query/stacks/fungible-tokens/fungible-token-metadata.query.ts

```diff
@@ -20,36 +21,38 @@ const queryOptions = {
   retry: 0,
 } as const;
 
-function fetchUnanchoredAccountInfo(client: StacksClient, limiter: RateLimiter) {
+function fetchUnanchoredAccountInfo(isTestnet: boolean, limiter: RateLimiter) {
   return (contractId: string) => async () => {
     await limiter.removeTokens(1);
-    return client.fungibleTokensApi.getContractFtMetadata({ contractId });
+    const resp = await fetcher(
+      `https://api.${isTestnet ? 'testnet' : 'mainnet'}.hiro.so/metadata/v1/ft/${contractId}`
+    );
```

Comment:
How can we do this in a way where we don't have to continually repeat this conditional when we do https://github.com/hirosystems/wallet/issues/3383